service/internal/resources.go

```diff
@@ -227,7 +227,7 @@ var _escData = map[string]*_escFile{
 		name:    "component_header.html",
 		local:   "templates/component_header.html",
 		size:    156,
-		modtime: 1594178791,
+		modtime: 1602111360,
```

Comment:
This file probably should not be changed in this PR.

Reply:
I'll remove it now